scripts/pyproject.toml.in

```diff
@@ -0,0 +1,187 @@
+# ``<package>`` if they exist.  The prefix(s) will be stripped from the package
+# name inside the wheel.
+# wheel.packages = ["<package>"]
+
+# The Python tags. The default (empty string) will use the default Python
+# version. You can also set this to "cp37" to enable the CPython 3.7+ Stable ABI
+# / Limited API (only on CPython and if the version is sufficient, otherwise
+# this has no effect). Or you can set it to "py3" or "py2.py3" to ignore Python
+# ABI compatibility. The ABI tag is inferred from this tag.
+wheel.py-api = "@PYPROJECT_PY_API@"
+
+# Fill out extra tags that are not required. This adds "x86_64" and "arm64" to
+# the list of platforms when "universal2" is used, which helps older Pip's
+# (before 21.0.1) find the correct wheel.
+wheel.expand-macos-universal-tags = false
+
+# The install directory for the wheel. This is relative to the platlib root. You
+# might set this to the package name. The original dir is still at
+# SKBUILD_PLATLIB_DIR (also SKBUILD_DATA_DIR, etc. are available). EXPERIMENTAL:
+# An absolute path will be one level higher than the platlib root, giving access
+# to "/platlib", "/data", "/headers", and "/scripts".
+wheel.install-dir = ""
+
+# A list of license files to include in the wheel. Supports glob patterns.
+wheel.license-files = ["LICEN[CS]E*", "ITK-source/ITK/LICENSE", "ITK-source/ITK/NOTICE"]
+
+# If set to True (the default), CMake will be run before building the wheel.
+wheel.cmake = true
+
+# Target the platlib or the purelib. If not set, the default is to target the
+# platlib if wheel.cmake is true, and the purelib otherwise.
+wheel.platlib = "@PYPROJECT_PLATLIB@"
+
+# A set of patterns to exclude from the wheel. This is additive to the SDist
+# exclude patterns. This applies to the final paths in the wheel, and can
+# exclude files from CMake output as well.  Editable installs may not respect
+# this exclusion.
+wheel.exclude = []
+
+# The build tag to use for the wheel. If empty, no build tag is used.
+wheel.build-tag = ""
+
+# If CMake is less than this value, backport a copy of FindPython. Set to 0
+# disable this, or the empty string.
+backport.find-python = "3.26.1"
+
+# Select the editable mode to use. Can be "redirect" (default) or "inplace".
+editable.mode = "redirect"
+
+# Turn on verbose output for the editable mode rebuilds.
+editable.verbose = true
+
+# Rebuild the project when the package is imported. The build-directory must be
+# set.
+editable.rebuild = false
+
+# The components to install. If empty, all default components are installed.
+install.components = []
+
+# Whether to strip the binaries. True for scikit-build-core 0.5+.
+install.strip = true
+
+# List dynamic metadata fields and hook locations in this table.
+metadata = {}
+
+# Strictly check all config options. If False, warnings will be printed for
+# unknown options. If True, an error will be raised.
+strict-config = true
+
+# Enable early previews of features not finalized yet.
+experimental = false
```

Comment:
The default on this will never change to true. :) I'd probably recommend not setting a bunch of things to their default values; we protect against changes with `minimum-version` already, and it hides the important parts.

Reply:
:+1: yes, trimmed.